src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -212,8 +218,31 @@ ss::future<> service::configure() {
           principal,
           security::acl_host::wildcard_host(),
           security::acl_operation::all,
-          security::acl_permission::allow}}},
-      5s);
+          security::acl_permission::allow}}};
+
+    auto create_acls_stop_time = model::time_from_now(30s);
+    while (err_vec[0] != cluster::errc::success
```

Comment:
issue: I think this still blocks shutdown if startup fails. Perhaps it worth checking the gate?